proxy/src/main/java/com/velocitypowered/proxy/VelocityServer.java

```diff
@@ -209,9 +209,8 @@ void start() {
 
     // Initialize commands first
     commandManager.register("velocity", new VelocityCommand(this));
-    commandManager.register("server", new ServerCommand(this));
-    commandManager.register("shutdown", ShutdownCommand.command(this),
-        "end", "stop");
+    commandManager.register("goto", new ServerCommand(this));
+    commandManager.register("shutdown", ShutdownCommand.command(this));
```

Comment:
This change is completely unnecessary and pointless as currently only the ConsoleCommandSource (the console) can execute the command, so there are no compatibility issues with Minecraft server commands

Reply:
Bruhh if give permission with lpv or alternative plugin for velocity permission yes